### PR TITLE
feature: add scripts/aux_swa.py

### PR DESCRIPTION
**Pull request by YuriCat**
Should we change the script name?

### CODE REVIEW

File: scripts/aux_swa.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+# SWA (running equal averaging)
+#
+
+model_dir = 'models'
+saved_model_path = os.path.join('models', 'swa.pth')
+
+ed, length = int(sys.argv[1]), int(sys.argv[2])
+step = 1
+if len(sys.argv) >= 4:
+    step
```
Comment (ikki407) [inline scripts/aux_swa.py:27]: ?

**Comment by ikki407** [comment]
I like just `swa.py` or `stochastic_weighted_averaging.py`.

**Comment by ikki407** [comment]
Do we need to update the parameters of batch normalization after weight parameter updates? If not, I think, the SWA model is saved with the BN parameters of the first loaded model, since the `swa_model.update_parameters()` doesn't update the BN params, right?

File: scripts/aux_swa.py (hunk 1 above)
Comment (YuriCat) [reply scripts/aux_swa.py]: fixed

**Pull request merged**